extensions/objects/templates/literally-canvas-diagram-editor.component.ts

```diff
@@ -53,12 +58,13 @@ angular.module('oppia').component('literallyCanvasDiagramEditor', {
       ctrl.getDiagramSizeInfo = function() {
         var maxWidth = MAX_DIAGRAM_WIDTH;
         var maxHeight = MAX_DIAGRAM_HEIGHT;
-        return 'This diagram has a maximum dimension of ' + maxWidth +
-        'px X ' + maxHeight + 'px to ensure that it fits in the card.';
+        return (
+          'This diagram has a maximum dimension of ' + maxWidth +
+          'px X ' + maxHeight + 'px to ensure that it fits in the card.');
       };
 
       ctrl.validate = function(data) {
-        // Will be implemented once data is saved.
+        // TODO(#9357): Will be implemented once data is saved.
```

Comment:
Could you please clarify what "once data is saved" means? Do you mean "once the functionality for saving XXX is implemented"?

Reply:
Yes that is exactly what I meant. I have changed it. Done.